src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -230,7 +230,11 @@ protected void consume(@NotNull PacketInfo packetInfo)
             });
         bitrateController = new BitrateController(this, diagnosticContext, logger);
 
-        messageTransport = new EndpointMessageTransport(this, logger);
+        messageTransport = new EndpointMessageTransport(
+            this,
+            () -> getConference().getVideobridge().getStatistics(),
```

Comment:
You can also just pass the reference

Reply:
How do you use method references with chained calls like that?